tests/integration/test_system_logs_recreate/test.py

```diff
@@ -33,7 +33,6 @@ def test_system_logs_recreate():
         "trace_log",
         "metric_log",
         "error_log",
-        "latency_log",
```

Comment:
Since this table is going away, should this change be marked backwards incompatible? For those who use this log table currently, it would be unavailable.